pandas/tests/frame/test_arithmetic.py

```diff
@@ -1577,3 +1577,13 @@ def test_arith_reindex_with_duplicates():
     result = df1 + df2
     expected = pd.DataFrame([[np.nan, 0, 0]], columns=["first", "second", "second"])
     tm.assert_frame_equal(result, expected)
+
+
+def test_arith_list_of_arraylike_raise():
+    # GH 36702. Raise when trying to add list of array-like to DataFrame
+    df = pd.DataFrame({"x": [1, 2], "y": [1, 2]})
+    ser = pd.Series([1, 1])
+
+    msg = f"Unable to coerce list of {type(ser)} to Series/DataFrame"
+    with pytest.raises(ValueError, match=msg):
```

Comment:
can you try the reverse op as well,  a single element and no elements in the list

Reply:
Added one-element list and reverse op to the test. Zero elements in the list throws "Unable to coerce to Series, length must be 2: given 0", and we already test for this error in `test_alignment_non_pandas`. I believe it's enough, but if you'd like to test for the error in the new test too, please tell me.